packages/dx-react-grid/docs/reference/virtual-table.md

```diff
@@ -16,6 +16,11 @@ Name | Type | Default | Description
 height | number | 530 | The virtual table's height.
 estimatedRowHeight | number | `37` for [Bootstrap3](https://www.npmjs.com/package/@devexpress/dx-react-grid-bootstrap3); `48` for [Material UI](https://www.npmjs.com/package/@devexpress/dx-react-grid-material-ui) | Estimated row height. Specify the average value for a table whose rows have different heights.
 columnExtensions? | Array&lt;[Table.ColumnExtension](table.md#tablecolumnextension)&gt; | Additional column properties that the plugin can handle.
+tableComponent | ReactNode | | A component that renders a table.
+headComponent | ReactNode | | A component that renders a table head.
+bodyComponent | ReactNode | | A component that renders a table body.
+stickyTableComponent | ReactNode | | A component that renders a sticky header.
```

Comment:
Fix description. And seems that the component name is not good enough.

Reply:
@kvet what about `stickyHeaderComponent`?

Reply:
"Fixed" is more appropriate term here. What about `fixedHeaderComponent`?